src/components/Collections/ListItems/PlioListItem.vue

```diff
@@ -219,25 +280,39 @@ export default {
         case "analyse":
           this.analysePlio();
           break;
+        case "delete":
+          // configure the dialog box
+          if (!this.dialogTitle) {
```

Comment:
mostly thought of it as an optimization - since no other method is setting the dialog title, we can only set it once and don't need to do it again and again - but that can also cause confusion - so, I'll remove it.

Reply:
Cool